packages/core/src/extensions/FormattingToolbar/FormattingToolbarPlugin.ts

```diff
@@ -221,15 +212,19 @@ export class FormattingToolbarProsemirrorPlugin extends EventEmitter<any> {
     this.plugin = new Plugin({
       key: formattingToolbarPluginKey,
       view: (editorView) => {
-        this.view = new FormattingToolbarView(editor, editorView, (state) => {
-          this.emit("update", state);
-        });
+        this.view = new FormattingToolbarView(
+          editor,
+          editorView,
+          (position) => {
+            this.emit("update position", position);
```

Comment:
To discuss on call:

naming (e.g.:  `updatePosition`), and theory about splitting all in 2 events